padertorch/contrib/tcl/utils/augmentation.py

```diff
@@ -0,0 +1,133 @@
+from scipy.signal import fftconvolve
+
+import paderbox as pb
+import lazy_dataset
+import numpy as np
+from typing import List, Tuple, Iterable, Union, Dict
+from functools import partial
+import padertorch as pt
+
+
+class AugmentationHelper:
+    def __init__(self,
+                 augmentation_sets: Dict = None,
+                 p_augment: float = 0.,
+                 p_reverb=None,
+                 augmentation_type: Union[str, Iterable] = ('noise', 'music', 'speech'),
```

Comment:
Add a hint, for the values? e.g. 'reverb seems to be allowed'

Reply:
Added a docstring